src/Pages/HomePage/components/CreateTask/CreateTask.tsx

```diff
@@ -26,6 +26,7 @@ function CreateTask({ opened, close }: CreateTaskProps) {
       description: data.description,
       dueDate: data.dueDate,
       priority: data.priority,
+      status: "initiated",
```

Comment:
Create a global constant file. Have this statuses declared as constants and reused them everywhere, instead of declaring magic strings everytime.

for e.g:

```
export const TASK_STATES = {
  ACTIVE_STATE: 'active',
  COMPLETED: 'completed',
};
```